splunk_event/test_splunk_event.py

```diff
@@ -993,6 +993,46 @@ def _mocked_saved_searches(*args, **kwargs):
         self.assertEquals(self.service_checks[0]['status'], 2, "service check should have status AgentCheck.CRITICAL")
 
 
+class TestSplunkSavedSearchesIgnoreError(AgentCheckTest):
+    """
+    Splunk event check should ignore exception when getting an exception from saved searches
+    """
+    CHECK_NAME = 'splunk_event'
+
+    def test_checks(self):
+        self.maxDiff = None
+
+        config = {
+            'init_config': {},
+            'instances': [
+                {
+                    'url': 'http://localhost:13001',
+                    'username': "admin",
+                    'password': "admin",
+                    'ignore_saved_search_errors': 'true',
```

Comment:
does value 'true' without the quotes also work? Maybe using the native boolean value may be cleaner than this string variant.

Reply:
changed to boolean value.